app/controllers/users_controller.rb

```diff
@@ -123,15 +124,21 @@ def destroy
   end
 
   def feedbacks
-    @invitations = Invitation.where(email: current_user.all_emails).includes(review: {associate_consultant: :user}).select do |i|
+    @invitations = Invitation.where(email: current_user.all_emails)
+                             .includes({ review: :associate_consultant },
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.